### PR TITLE
Escape added definition paths

### PR DESCRIPTION
**Pull request by bergundy**

### CODE REVIEW

File: src/lib/common.ts [hunk 1]
```diff
@@ -46,7 +47,7 @@ export function createClassValidator(
 ): ClassValidator {
   const ajv = createValidator();
   for (const [k, v] of Object.entries(schema.definitions)) {
-    ajv.addSchema(v, `#/definitions/${k}`);
+    ajv.addSchema(v, `#/definitions/${url.format(k)}`);
```
Comment (arielshaqed) [inline src/lib/common.ts]: I'm pretty sure this is not a safe way to escape weird TypeScript characters inside a weirder [JSON pointer](https://tools.ietf.org/html/rfc6901).  Perhaps https://www.npmjs.com/package/json-pointer-rfc6901#escapestr---str or a similar method would be more reliable?
Failing that, please name things to make it clear what's going on, e.g.
```ts
const escapeJsonPointer = url.format;
```

File: src/test/test_rpc.ts [hunk 2]
```diff
@@ -1019,3 +1019,37 @@ export default async function test(client: TestClient) {
 `;
   await new TestCase(schema, handler, tester).run();
 });
+
+test('rpc works for Partial param types', pass, async () => {
```
Comment (arielshaqed) [inline src/test/test_rpc.ts:1023]: Do we also get a good TypeScript type after going there and back?  (May I hope it's a _reasonable_ type??)

**Review by arielshaqed — COMMENTED** [review]
Thanks!  Not sure this is the way to escape characters.

File: src/test/test_rpc.ts (hunk 2 above)
Comment (bergundy) [reply src/test/test_rpc.ts:1023]: It becomes `Partial_of_User_end` which is acceptable.
We're going to have to rewrite concord at some point and avoid the typescript->json->typescript conversion.

File: src/lib/common.ts (hunk 1 above)
Comment (bergundy) [reply src/lib/common.ts]: Addressed.

**Review by arielshaqed — APPROVED** [review]
Thanks!

**Pull request merged**